### DISCUSSION TITLE
Ruleset Leaderboards

### DISCUSSION THREAD

**Discussion by BowiePro** [ppy/osu#15157] [Ideas] [answered]
I propose a feature where the creator of a ruleset can upload it to a "ruleset hub" where people can download it, and it gets a leaderboard if there are enough active players. A pp system is integrated into the ruleset file if the creator chooses. Otherwise, the leaderboards are by score. The creator might even be able to add beatmap nominators for their ruleset, although problems could arise from that. If the ruleset has a leaderboard, it would be listed on a player's profile in an "other" section, given the player has played it. A map with difficulties in official and custom rulesets would not be able to get ranked, but would be able to get loved.

**Comment by peppy** [comment, marked as answer]
as you may expect, that is the eventual plan. many other things to do first.

**Reply by rasputozen** [reply]
Couldn't that implementation fracture the playerbase though? Do you not think it's possible to have each mod's score multiplier accurately reflect the added difficulty it imposes on a map?

**Reply by peppy** [reply]
it seems like you're in the wrong place.

**Reply by rasputozen** [reply]
I'm genuinely interested to hear your thoughts on those questions. Where would you prefer me to ask you them?

**Reply by peppy** [reply]
I have no idea what you're asking, but open a new discussion or ask somewhere else, like at a community meeting.

**Reply by integerrr** [reply]
@rasputozen I think you've confused "ruleset" for "mods", you might see others refer to rulesets as "gamemodes". This post is about leaderboards for new rulesets/gamemodes people come up with for lazer, not mods

**Reply by rasputozen** [reply]
Ah that's exactly what I did! I never came across the term "ruleset" in osu and figured the OP was trying to describe specific mod configurations. From that I deduced separate leaderboards for various mod combos lol. My mistake.
